app/models/form526_submission.rb

```diff
@@ -474,6 +474,21 @@ def account
     Account.lookup_by_user_uuid(user_uuid)
   end
 
+  # Send the Submitted Email - when the Veteran has clicked the "submit" button in va.gov
+  # Primary Path: when the response from getting a claim id is successful
+  # Backup Path: when SubmitForm526 Job has exhausted,
+  # or when we get a non_retryable_error response from claim establishment flow
+  # @param invoker: string where the Received Email trigger is being called from
+  def send_submitted_email(invoker)
+    if Flipper.enabled?(:disability_526_send_form526_submitted_email)
+      Rails.logger.info("Form526SubmittedEmailJob called for user #{user_uuid},
+                                                          submission: #{id} from #{invoker}")
```

Comment:
double checking–`uuid` is not PII, right?

Reply:
afaik, it's already in Datadog logs!